src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -29,23 +31,47 @@
 public class MediaStreamTrackDesc
 {
     /**
+     * The system property name that holds the minimum time (in millis) that is
+     * required for the media engine to generate a new key frame.
```

Comment:
ah i see.  so the client-side stack won't generate them more frequently than that, so we don't bother requesting.  i think it's worth documenting that, especially since that's technically chrome-specific, right?  i think the variable name is ok as is, just updating the comment a bit to clarify.  i was just confused by 'generate' since we don't generate key frames here, just request them.